src/__tests__/layout/LeftSideBar.test.tsx

```diff
@@ -1,8 +1,17 @@
 import React from 'react';
 import { render } from '@testing-library/react';
+import type { LinkProps } from 'next/link';
 
 import LeftSidebar from '@/layout/LeftSidebar';
 
+jest.mock('next/link', () => jest.fn(
+  (
+    props: LinkProps & { children?: React.ReactNode } & React.HTMLAttributes<HTMLAnchorElement>,
+  ) => (
+    <a className={props.className} href={props.href.toString()}>{props.children}</a>
+  ),
+));
```

Comment:
unrelated but deletes a warning of missing `act` due to async update of `ForwardRef(LinkComponent)`.